### DISCUSSION TITLE
EPERM: operation not permitted, unlink

### DISCUSSION THREAD

**Discussion by TheHolyRoller** [prisma/prisma#21869] [Q&A]
### Bug description

Finished setting up my Prisma schema and loaded in my environment variables. Entered npx prisma db push and got the error. 

`Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": MySQL database "twitter1" at "aws.connect.psdb.cloud"

The database is already in sync with the Prisma schema.    

EPERM: operation not permitted, unlink
'C:\Users\User\Documents\Twitter\twitter\node_modules\.pris
ma\client\query_engine-windows.dll.node'
`

### Expected behavior

Script Finished Correctly 

### Prisma information

All the prisma configurations are the ones taken from the guide.

### Environment & setup

    OS: Windows 11
    Database: PlanetScale latest version
    Node.js version: 18.18.2 using npm 

### Prisma Version

```
{
  "name": "twitter",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "build": "next build",
    "db:push": "prisma db push",
    "db:studio": "prisma studio",
    "dev": "next dev",
    "postinstall": "prisma generate",
    "lint": "next lint",
    "start": "next start"
  },
  "dependencies": {
    "@next-auth/prisma-adapter": "^1.0.7",
    "@prisma/client": "^5.5.2",
    "@t3-oss/env-nextjs": "^0.7.0",
    "@tanstack/react-query": "^4.32.6",
    "@trpc/client": "^10.37.1",
    "@trpc/next": "^10.37.1",
    "@trpc/react-query": "^10.37.1",
    "@trpc/server": "^10.37.1",
    "next": "^14.0.0",
    "next-auth": "^4.23.0",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "superjson": "^1.13.1",
    "zod": "^3.22.4"
  },
  "devDependencies": {
    "@types/eslint": "^8.44.2",
    "@types/node": "^18.16.0",
    "@types/react": "^18.2.33",
    "@types/react-dom": "^18.2.14",
    "@typescript-eslint/eslint-plugin": "^6.3.0",
    "@typescript-eslint/parser": "^6.3.0",
    "autoprefixer": "^10.4.14",
    "eslint": "^8.47.0",
    "eslint-config-next": "^14.0.0",
    "postcss": "^8.4.27",
    "prettier": "^3.0.0",
    "prettier-plugin-tailwindcss": "^0.5.1",
    "prisma": "^5.5.2",
    "tailwindcss": "^3.3.3",
    "typescript": "^5.1.6"
  },
  "ct3aMetadata": {
    "initVersion": "7.23.1"
  },
  "packageManager": "npm@9.8.1"
}

```

**Comment by ludralph** [comment]
Hi @TheHolyRoller 👋 

Thank you for raising this question.

Can you please try the suggestions in this [StackOverflow](https://stackoverflow.com/questions/46020018/error-eperm-operation-not-permitted-unlink-d-sources-node-modules-fseven) post?

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ayushpatil2122** [comment]
To solve this issue close user server first and then run a command

**Reply by ZerubbabelT** [reply]
Thank u so muchh

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
